doc/design-3.0.rst

```diff
@@ -2,10 +2,24 @@
 Ganeti 3.0 design
 =================
 
-The following designs have been partially implemented in Ganeti 3.0.
+This document describes the major changes between Ganeti 2.16 and 3.0.
 
-- none
+Objective
+=========
 
-The following designs' implementations were completed in Ganeti 3.0.
+All Ganeti versions prior to 3.0 are built on Python 2 code. With Python 3 being the offical default and version 2 slowly fading out of Linux distributions, the goal of this major release is to achieve full and only Python 3 compatibility.
 
-- none
+Minimum supported Python version
+================================
+
+We require a minimum of Python 3.6 for the current code to work. This allows us to support stable distributions like Debian Buster or Ubuntu 18.04 LTS and to create working upgrade paths for our users.
+
+What happened to Ganeti 2.17?
+=============================
+
+A beta version of 2.17 has been released on February 22nd, 2016. This version has not seen exhaustive testing but included a number of new features. To push forward with the migration to Python 3, we decided to ditch these changes and start over based on Ganeti 2.16. Since the upgrade to 2.17.0~beta1 introduced some breaking changes, there is currently no direct upgrade path between 2.17 and 3.0. The only known but untested way is to downgrade to Ganeti 2.16 first and then upgrade to 3.0. This will remove access to uncomplete features like the Ganeti Maintenance Daemon.
```

Comment:
just minor spelling: either "incomplete" or "uncompleted" instead of "uncomplete"

Otherwise LGTM.

Reply:
Sorry, overlooked that. That is now fixed, thanks :-)